crates/data_components/src/duckdb/creator.rs

```diff
@@ -242,6 +243,9 @@ impl TableCreator {
             )
             .context(super::UnableToQueryDataSnafu)?;
 
+        // DuckDB doesn't add IF NOT EXISTS to CREATE TABLE statements, so we add it here.
+        let create_stmt = create_stmt.replace("CREATE TABLE", "CREATE TABLE IF NOT EXISTS");
```

Comment:
Please review ig there could be a case where "IF NOT EXISTS" already there so it will end up with "CREATE TABLE IF NOT EXISTS IF NOT EXISTS" if there is already

Reply:
DuckDB doesn't return if not exists. If that changed one of the tests I added would fail